### ISSUE TITLE
Cargo install fails

### ISSUE THREAD

**Issue by CaptainQuirk** [ivanceras/svgbob#31]
When trying to install with **cargo**, I get the following error

```    
Updating registry `https://github.com/rust-lang/crates.io-index`
  Installing svgbob_cli v0.3.0
   Compiling libc v0.2.43
   Compiling unicode-width v0.1.5
   Compiling vec_map v0.8.1
   Compiling ansi_term v0.11.0
   Compiling bitflags v1.0.4
   Compiling pom v1.1.0
   Compiling svg v0.5.11
   Compiling strsim v0.7.0
   Compiling textwrap v0.10.0
   Compiling atty v0.2.11
   Compiling clap v2.32.0
   Compiling svgbob v0.3.0
error[E0433]: failed to resolve. Use of undeclared type or module `svg`
   --> /home/leonard/.cargo/registry/src/github.com-1ecc6299db9ec823/svgbob-0.3.0/src/element.rs:420:33
    |
420 |                 let text_node = svg::node::Text::new(string.to_string());
    |                                 ^^^ Use of undeclared type or module `svg`

error[E0433]: failed to resolve. Use of undeclared type or module `pom`
   --> /home/leonard/.cargo/registry/src/github.com-1ecc6299db9ec823/svgbob-0.3.0/src/grid.rs:636:23
    |
636 | fn escape_string() -> pom::parser::Parser<'static, char, (usize, usize)> {
    |                       ^^^ Use of undeclared type or module `pom`

error[E0433]: failed to resolve. Use of undeclared type or module `pom`
   --> /home/leonard/.cargo/registry/src/github.com-1ecc6299db9ec823/svgbob-0.3.0/src/grid.rs:643:20
    |
643 | fn line_parse() -> pom::parser::Parser<'static, char, Vec<(usize, usize)>> {
    |                    ^^^ Use of undeclared type or module `pom`

error[E0554]: #![feature] may not be used on the stable release channel
  --> /home/leonard/.cargo/registry/src/github.com-1ecc6299db9ec823/svgbob-0.3.0/src/lib.rs:33:1
   |
33 | #![feature(extern_prelude)]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: aborting due to 4 previous errors

Some errors occurred: E0433, E0554.
For more information about an error, try `rustc --explain E0433`.
error: Could not compile `svgbob`.
warning: build failed, waiting for other jobs to finish...
error: failed to compile `svgbob_cli v0.3.0`, intermediate artifacts can be found at `/tmp/cargo-installOjHfGN`

Caused by:
  build failed
```

**Comment by ivanceras** [comment]
Thanks for reporting, I just published [`0.3.1`](https://crates.io/crates/svgbob_cli)  that can be build with the stable compiler. Please try again.

**Comment by CaptainQuirk** [comment]
Great, testing it happily since yesterday ! Thanks
